service/controller/v5/cloudconfig/template.go

```diff
@@ -5,8 +5,10 @@ const (
 	calicoAzureFileOwner      = "root:root"
 	calicoAzureFilePermission = 0600
 	calicoAzureFileTemplate   = `# Extra changes:
-#  - added "nodename_file_optional" set to true (can be removed on the next upgrade)
-#  - added resource limits
+#  - Added "nodename_file_optional" set to true (can be removed on the next upgrade).
```

Comment:
Do we track the removal? I recall we had an issue for this. Then we maybe should align with the TODO issue link thingy here as well.

Reply:
Yes we have a task and we tracking this in team magic board. I'm taking care in few months after calico 3.2.x will become active to make sure all clusters have 3.2 (and kubelet with proper mount.)

https://github.com/giantswarm/giantswarm/issues/4113


i'll add